COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -876,7 +884,7 @@ namespace COLLADASaxFWL
     }
 
     //------------------------------
-    void MeshLoader::initializeColorsOffset ()
+    bool MeshLoader::initializeColorsOffset ()
```

Comment:
I that that color is as optional as normals, tangents, binormals. I don't see why this is coded as required data.